src/NUnitFramework/nunit.framework.legacy/nunit.framework.legacy.csproj

```diff
@@ -3,7 +3,6 @@
   <PropertyGroup>
     <TargetFrameworks>$(NUnitLibraryFrameworks)</TargetFrameworks>
     <RootNamespace>NUnit.Framework.Legacy</RootNamespace>
-    <NoWarn/>
```

Comment:
Reset this as now it inherits from Directory.Build.props which as a NoWarn of 1591 we use for our test projects.

Reply:
Done.